src/gui/mzroll/tabledockwidget.cpp

```diff
@@ -2827,6 +3658,7 @@ void ScatterplotTableDockWidget::setupPeakTable() {
     colNames << "Max quality";
     colNames << "MS2 score";
     colNames << "#MS2 events";
+    colNames << "Probability"; // TODO: add this column conditionally
```

Comment:
Don't you think we should name this "Classification probability" instead? Ignore if this decision was made after discussion already.

Reply:
Sure..Will discuss it with Richa and change it accordingly.